src/libraries/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
According to `.editorconfig`, `.csproj` files should be UTF-8 without byte-order mark. Which editor inserted this?

Reply:
https://github.com/dotnet/runtime/pull/119310#discussion_r2319511306

I don't know if this may help: https://github.com/editorconfig/editorconfig-notepad-plus-plus